beringei/tools/CMakeLists.txt

```diff
@@ -56,3 +56,4 @@ target_link_libraries(
 )
 
 add_subdirectory(grafana_read_service)
+add_subdirectory(plain_text_service)
```

Comment:
I think than mixing grafana and plain text protocol can be confusing.
But I can merge then if you prefer :)
Should I rename grafana_read_service in the same time?